oraclizeAPI_0.4.25.sol

```diff
@@ -362,157 +399,1290 @@ contract usingOraclize {
         return false;
     }
 
-    function __callback(bytes32 myid, string result) public {
+    function __callback(
+        bytes32 myid,
+        string result
+    )
+        public
+    {
         __callback(myid, result, new bytes(0));
     }
-    function __callback(bytes32 myid, string result, bytes proof) public {
-      return;
-      // Following should never be reached with a preceding return, however
-      // this is just a placeholder function, ideally meant to be defined in
-      // child contract when proofs are used
-      myid; result; proof; // Silence compiler warnings
-      oraclize = OraclizeI(0); // Additional compiler silence about making function pure/view. 
-    }
 
-    function oraclize_getPrice(string datasource) oraclizeAPI internal returns (uint){
-        return oraclize.getPrice(datasource);
+    function __callback(
+        bytes32 myid,
+        string result,
+        bytes proof
+    )
+        public
+    {
+        return;
+        // Following should never be reached with a preceding return, however
+        // this is just a placeholder function, ideally meant to be defined in
+        // child contract when proofs are used
+        myid; result; proof; // Silence compiler warnings
+        oraclize = OraclizeI(0); // Additional compiler silence about making function pure/view.
     }
-
-    function oraclize_getPrice(string datasource, uint gaslimit) oraclizeAPI internal returns (uint){
-        return oraclize.getPrice(datasource, gaslimit);
+    /**
+     *
+     * @notice oraclize_getPrice(...) overloads follow...
+     *
+     */
+    function oraclize_getPrice(
```

Comment:
wouldn't it be useful to specify these as view?

Reply:
Oops - remembered the less obvious bit of making it all view inside the interfaces but not here! Fixed!

Reply:
correction to my request, due to oraclizeAPI actually having the potential to modify state, we've dropped this.